marklogic-data-hub/src/test/java/com/marklogic/hub/HubTestBase.java

```diff
@@ -515,9 +523,11 @@ public void createProjectDir(String projectDirName) {
             if ( finalTimestampDirectory.exists() ) {
                 FileUtils.forceDelete(finalTimestampDirectory);
             }
-            Path devProperties = Paths.get(".").resolve("gradle.properties");
-            Path projectProperties = projectDir.toPath().resolve("gradle.properties");
-            FileUtils.copyFile(devProperties.toFile(), projectProperties.toFile());
+            File projectProperties = projectDir.toPath().resolve("gradle.properties").toFile();
+            if(!projectProperties.exists()) {
+                projectProperties.createNewFile();
+            }
+            properties.store(new FileOutputStream(projectProperties), "");
```

Comment:
Can we have both gradle.properties and gradle-env.properties in the project folder rather than one gradle.properties file with values overridden by gradle-env.properties?

Reply:
Addressed the changes mentioned in the commit "Bootstrap task recognizes the environment"
e6e6d8f